request/graphql/schema/types/types.go

```diff
@@ -127,6 +130,20 @@ var (
 		},
 	})
 
+	// CRDTFieldDirective @crdt is used to define the CRDT type of a field
+	CRDTFieldDirective *gql.Directive = gql.NewDirective(gql.DirectiveConfig{
+		Name:        CRDTDirectiveLabel,
+		Description: crdtDirectiveDescription,
+		Args: gql.FieldConfigArgument{
+			CRDTDirectivePropType: &gql.ArgumentConfig{
+				Type: gql.String,
```

Comment:
question: Would it be possible to make this an enum type that consists of all of our supported CRDT types?

Reply:
Made the change to enum. Let me know what you think.